peer_review_data/main.py

```diff
@@ -99,103 +96,116 @@ def saveAssessmentsAndComments(
+                        canvasComment, assessment)
+                LOGGER.debug(f'Saving {comment}…')
+                comment.save()
+            except Exception as e:
+                LOGGER.warning(f'Error saving {comment}: {e}')
 
 
 def main() -> None:
     timeStart: datetime = datetime.now(tz=utc)
     LOGGER.info(f'Start time: {timeStart.isoformat(timespec="milliseconds")}')
 
-    canvasCourse: CanvasCourse = canvas.get_course(COURSE_ID)
-    LOGGER.info(f'Found course ({canvasCourse.id}): "{canvasCourse.name}"')
-    courseSaved = False
-    course: models.Course | None = None
-    user: models.User | None = None
-
-    canvasAssignment: CanvasAssignment = canvasCourse.get_assignment(
-        ASSIGNMENT_ID)
-    LOGGER.info(
-        f'Found assignment ({canvasAssignment.id}): "{canvasAssignment.name}"')
-
-    if canvasAssignment.peer_reviews is not True:
-        LOGGER.info(
-            f'Skipping assignment ({ASSIGNMENT_ID}) in course ({COURSE_ID}): '
-            'Not configured for peer reviews.')
-        sys.exit()
-
-    LOGGER.info(f'Assignment ({canvasAssignment.id}) is peer reviewed')
-
-    assignmentRubricId: int = canvasAssignment.rubric_settings.get('id')
-    LOGGER.info(f'Assignment ({canvasAssignment.id}) has '
-                f'rubric ID ({assignmentRubricId})')
-
-    canvasAssignmentRubric: CanvasRubric = canvasCourse.get_rubric(
-        assignmentRubricId, include='assessments', style='full')
-
-    if not hasattr(canvasAssignmentRubric, 'assessments'):
-        LOGGER.info(f'Skipping assignment ({ASSIGNMENT_ID}) in '
-                    f'course ({COURSE_ID}): Not configured for peer reviews '
-                    '("assessments").')
-        sys.exit()
-
-    LOGGER.info(f'Assignment ({ASSIGNMENT_ID}) in course ({COURSE_ID}) is '
-                'configured for peer reviews ("assessments")…')
-
-    if len(canvasAssignmentRubric.assessments) == 0:
-        LOGGER.info(
-            f'Skipping assignment ({ASSIGNMENT_ID}) in course ({COURSE_ID}): '
-            'No peer reviews ("assessments") were found.')
-        sys.exit()
-
-    LOGGER.info(f'Assignment ({ASSIGNMENT_ID}) in course ({COURSE_ID}) has '
-                'peer reviews ("assessments")…')
-
-    if not courseSaved:
-        courseSaved, course = saveCourseAndUsers(canvasCourse)
-
-    assignment: models.Assignment = models.Assignment.fromCanvasAssignment(
-        canvasAssignment)
-    LOGGER.info(f'Saving {assignment}…')
-    assignment.save()
-
-    LOGGER.info(f'Saving submissions for {assignment}…')
-    saveSubmissions(canvasAssignment)
-
-    LOGGER.info('Saving rubric and criteria…')
-    rubric, criteria = saveRubricAndCriteria(canvasAssignmentRubric,
-                                             canvasAssignment)
-
-    LOGGER.info('Saving assessments and comments…')
-    saveAssessmentsAndComments(canvasAssignmentRubric.assessments)
+    courseId: str
+    for courseId in [c.strip() for c in config.COURSE_IDS_CSV.split(',')]:
```

Comment:
You might consider doing the string splitting and stripping earlier, maybe just do it it `config`.

Reply:
True.  It'd be neater if I do it in `config`.

Reply:
Done.  Resolved in 2edf11d.